segmentstore/server/host/src/main/java/io/pravega/segmentstore/server/host/handler/TrackedConnection.java

```diff
@@ -66,6 +68,7 @@ void adjustOutstandingBytes(int delta) {
      */
     void send(WireCommand cmd) {
         this.connection.send(cmd);
+        log.debug("Sent response: {}", cmd);
```

Comment:
wouldnt logging it here result in a flood of logs for the append path/connection hello's etc.

Reply:
It's debug. So that's fine.